### DISCUSSION TITLE
color glitch

### DISCUSSION THREAD

**Discussion by Crepo920** [Godiesc/firefox-gx#224] [Q&A]
when i enter firefox it logo stays Lightgreen but when i open a new Tab it Become red(fushia)  how can i fix it? i tryed changing themes but it only work for 1st tab.As soon as i open a new tab the logo be come red. heres a video of my problem.

https://github.com/user-attachments/assets/c16121da-ae47-424b-af51-7250b76aacdc

**Comment by Godiesc** [comment]
Hi, for each wallpaper you have an accent color, in that file you can see.

**Reply by Godiesc** [reply]
in the `ogx_root-personal.css` you have the variable to change the color:

![imagen](https://github.com/user-attachments/assets/cc7c9d69-3a98-485a-9f0d-0012f940114d)
